integration_test/testgenerated/server/server_test.go

```diff
@@ -0,0 +1,174 @@
+package server_test
+
+import (
+	"context"
+	"errors"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+
+	"github.com/palantir/pkg/bearertoken"
+	"github.com/palantir/pkg/datetime"
+	"github.com/palantir/pkg/rid"
+	"github.com/palantir/pkg/safelong"
+	"github.com/palantir/pkg/uuid"
+	"github.com/palantir/witchcraft-go-server/wrouter"
+	"github.com/palantir/witchcraft-go-server/wrouter/whttprouter"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/palantir/conjure-go/v5/integration_test/testgenerated/server/api"
+)
+
+func TestSafeMarker(t *testing.T) {
+	router := wrouter.New(whttprouter.New())
+	err := api.RegisterRoutesTestService(router, testServerImpl{})
+	require.NoError(t, err)
+	called := false
+	router.AddRouteHandlerMiddleware(func(rw http.ResponseWriter, r *http.Request, reqVals wrouter.RequestVals, next wrouter.RouteRequestHandler) {
+		if reqVals.Spec.PathTemplate == "/safe/{myPathParam1}/{myPathParam2}" {
```

Comment:
note: It would be nice if we generated these as const values in the server file so we wouldn't have to copy+paste. This comes up anytime we need to implement route middleware.